src/main/java/com/princexml/wrapper/AbstractPrince.java

```diff
@@ -845,15 +851,17 @@ public void setOwnerPassword(String ownerPassword) {
     }
 
     /**
-     * Disallow printing of the PDF file.
+     * Disallow printing of the PDF file. Enabled by default unless explicitly
+     * disabled.
```

Comment:
Now that I think about it enabled/disabled/allowed/disallowed is really confusing here! They have to pass "true" to disallow printing, and the parameter is false by default, hmm.

Reply:
Yeah, it's a 1-to-1 mapping of the command line options. The existing C# and PHP wrappers also "suffer" from a similar issue, however some have been switched around (e.g. embed/subset fonts).

Would you like this changed so that there are no negatives in the method names?

Reply:
I think it's reasonable to have a negative in the method names, given that allow _is_ the default, but perhaps we can just be clearer that the default value is `false` rather than saying disabled/enabled?

The other option would be to make it `Permission.DISALLOWED` or something but that sounds like it could be more trouble than it's worth.

Reply:
Oh right, gotcha. Sure, I'll update the docs.